starlette/README.md

```diff
@@ -0,0 +1,42 @@
+# Starlette Temporal Workflow Template
+
+This repository provides settings for [Starlette](https://www.starlette.io/) and the Temporal Python SDK for durable workflows.
```

Comment:
Question now that tests are gone and all this does is set a client and then call Temporal on route - is there any value in the sample? Can't anyone with any framework create a Temporal client and then use it in a handler? I don't mind adding it, but it seems unnecessary to build this same thing for every framework that does the same thing.